api/models/task.py

```diff
@@ -695,7 +706,11 @@ def getWithRoundAndMetricMetadata(self, task_id_or_code):
                             # TODO: make the frontend use pretty_name?
                             "field_name": field_name,
                             "default_weight": self.get_default_metric_weight(
-                                t, field_name, t_dict["perf_metric_field_name"]
+                                field_name,
+                                t_dict["perf_metric_field_name"],
+                                annotation_config["aggregation_metric"][
+                                    "constructor_args"
+                                ].get("default_weights", {-1: -1}),
```

Comment:
I assume the default here will never actually be used since `field_name` should never be the integer `-1`, but since the default seems to be 1, could we change it to {-1: 1}